java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java

```diff
@@ -322,6 +324,7 @@ private StringBuilder getOutputForSingleNumber(
     } catch (NumberParseException e) {
       output.append(e.toString());
```

Comment:
The error string comes from a fixed set of strings, btw, so there's no danger of something unexpected there.